Tetris_formal.py

```diff
@@ -95,47 +139,34 @@ def get_direction(self, color):
             [color, self.x - 40, self.y - 40],
         ]
         return up, down, left, right
-    
-    
+
     def get_color(self):
-        num = random.randint(1,2)
-        
-        if num == 1 :
-            color = (70,90,120)
-        
-        elif num == 2 :
-            
-            color = pygame.image.load('/Users/coolguy/Documents/picture_for_L.jpg')
-            color = pygame.transform.scale(color,(40,40))
-            color.blit(color,(self.x, self.y))
-            
-       
+        """
```

Comment:
Ditto. The description doesn't mean anything. Please briefly describe the function.